src/scenes/AcceleratorRouter/ParticipantProjects/ParticipantProjectProvider.tsx

```diff
@@ -30,7 +30,7 @@ const ParticipantProjectProvider = ({ children }: Props) => {
   const { activeLocale } = useLocalization();
   const { project, projectId } = useProjectData();
   const { participant } = useParticipant(project?.participantId ?? -1);
-  const { acceleratorOrgs, reload: reloadAll } = useAcceleratorOrgs(true);
+  const { acceleratorOrgs, reload: reloadAll } = useAcceleratorOrgs({ hasProjectApplication: true });
```

Comment:
This provider is only used in the single and edit project view, in the admin console. In this location, we need to find all organizations that contain a project with an application, because they are not guaranteed to have an "accelerator" internal tag nor a project that is associated to a participant.